.licensed.yaml

```diff
@@ -0,0 +1,24 @@
+allowed:
+  - mit 
+  - apache-2.0
+  - bsd-2-clause
+  - bsd-3-clause
+  - isc
+  - mpl-2.0         # Needed for hashicorp stuff
+
+
+reviewed:
+  go:
```

Comment:
Do we need these listed here? From the documentation [here](https://github.com/github/licensed/blob/master/docs/configuration.md#applications), it sounds like this area is for dependencies that have licenses not in the allowed list, but all of these don't follow that guidance?

Reply:
I'll try removing them & see if v3 does a better job resolving the licenses.  These all showed up as `other` & failed the build.

Reply:
Oh, strange...